src/api/edge/imagesInfo.js

```diff
@@ -0,0 +1,116 @@
+export const EDGE_API = '/api/edge/v1';
+import { instance } from '@redhat-cloud-services/frontend-components-utilities/interceptors/interceptors';
+
+export const getImageSet = ({
+  id,
+  q = {
+    limit: 10,
+    offset: 0,
+    sort_by: '-created_at',
+  },
+}) => {
+  const query = getTableParams(q);
+  return instance.get(`${EDGE_API}/image-sets/${id}?${query}`);
+};
+
+export const getImageSetViewVersions = ({
+  imageSetID,
+  query = {
+    limit: 20,
+    offset: 0,
+    sort_by: '-created_at',
+  },
+}) => {
+  const q = getTableParams(query);
+  return instance.get(
+    `${EDGE_API}/image-sets/view/${imageSetID}/versions?${q}`
+  );
+};
+
+export const getImageSetView = ({ id }) => {
+  return instance.get(`${EDGE_API}/image-sets/view/${id}`);
+};
+
+export const getImagePackageMetadata = (id) => {
+  try {
+    return instance.get(`${EDGE_API}/images/${id}/metadata`);
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+export const fetchEdgeImages = (
+  q = {
+    limit: 20,
+    offset: 0,
+    sort_by: '-created_at',
+  }
+) => {
+  const query = getTableParams(q);
+  return instance.get(`${EDGE_API}/images?${query}`);
+};
+
+export const fetchEdgeImageSets = (
+  q = {
+    limit: 20,
+    offset: 0,
+    sort_by: '-created_at',
+  }
+) => {
+  const query = getTableParams(q);
+  return instance.get(`${EDGE_API}/image-sets?${query}`);
+};
+
+export const getEdgeImageStatus = (id) => {
+  return instance.get(`${EDGE_API}/images/${id}/status`);
+};
+
+export const getImageDataOnDevice = (id) => {
+  return instance.get(`${EDGE_API}/updates/device/${id}/image`);
+};
+
+export const getImageById = ({ id }) => {
+  return instance.get(`${EDGE_API}/images/${id}/details`);
+};
+
+export const getImageSets = ({ query }) => {
+  if (query === '') {
+    query = { limit: 20, offset: 0, sort_by: '-updated_at' };
+  }
+  const q = getTableParams(query);
```

Comment:
Maybe instead of 'q' this is named 'params'. same thing with the prop passed in on line 53